cellarium/ml/core/module.py

```diff
@@ -10,23 +10,31 @@
 import torch
 from lightning.pytorch.utilities.types import STEP_OUTPUT, OptimizerLRSchedulerConfig
 
+from cellarium.ml.core.datamodule import CellariumAnnDataDataModule, collate_fn
 from cellarium.ml.core.pipeline import CellariumPipeline
 from cellarium.ml.models import CellariumModel
-from cellarium.ml.utilities.core import copy_module
+from cellarium.ml.utilities.core import FunctionComposer, copy_module
 
 
 class CellariumModule(pl.LightningModule):
     """
     ``CellariumModule`` organizes code into following sections:
 
+        * :attr:`cpu_transforms`: A list of transforms to apply to the input data as part of the dataloader on CPU.
         * :attr:`transforms`: A list of transforms to apply to the input data before passing it to the model.
+        * :attr:`module_pipeline`: A :class:`cellarium.ml.core.CellariumPipeline` to apply all transforms,
+          minus the CPU transforms if they are handled by a :class:`CellariumAnnDataDataModule`, and the model.
         * :attr:`model`: A :class:`cellarium.ml.models.CellariumModel` to train with
           :meth:`training_step` method and epoch end hooks.
         * :attr:`optim_fn` and :attr:`optim_kwargs`: A Pytorch optimizer class and its keyword arguments.
         * :attr:`scheduler_fn` and :attr:`scheduler_kwargs`: A Pytorch lr scheduler class and its
           keyword arguments.
 
     Args:
+        cpu_transforms:
```

Comment:
Please write some notes about the logic behind CPU transforms under a "Notes" section at the end of the docstring.